tests/baselines/reference/objectLiteralShorthandPropertiesErrorFromNotUsingIdentifier.js

```diff
@@ -35,7 +35,8 @@ var y = {
     "typeof": 
 };
 var x = (_a = {
-        a: a, : .b,
+        a: a,
+        : .b,

```

Comment:
It looks like these baseline changes are due to us no longer checking if previous and next nodes (when emitting the node list) are on the same line if either one does not have a parent set. We only check if nodes are on the same line if they have the same parent container. 

It looks like the logic for checking if there is a parent set has been introduced as part of the following refactoring (not sure if this was intentional; or if it's reasonable to continue doing that). https://github.com/Microsoft/TypeScript/commit/68b0323b72bf51236d8cd181ed94ba9a63f933fd.